src/desktop/ApplicationWindow.js

```diff
@@ -143,6 +143,7 @@ export class ApplicationWindow {
 			show: false,
 			autoHideMenuBar: true,
 			webPreferences: {
+				partition: "nopersist",
```

Comment:
I think this would make each session in-memory only and all storage (including `localStorage`) will be cleared on every window close/open.

Reply:
According to my tests, it seems to be cleared on every restart (not every window close/open), which is why I didn't notice the login creds went missing. That's no good, so this needs a little more work.